src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -648,6 +648,33 @@
   <data name="GeneralPage_CheckForUpdates.Content" xml:space="preserve">
     <value>Check for updates</value>
   </data>
+  <data name="General_SettingsBackupAndRestoreLocationText.Description" xml:space="preserve">
+    <value />
+  </data>
+  <data name="General_SettingsBackupAndRestoreLocationText.Header" xml:space="preserve">
+    <value>Location</value>
+  </data>
+  <data name="General_SettingsBackupAndRestoreStatusInfo.Description" xml:space="preserve">
+    <value />
+  </data>
+  <data name="General_SettingsBackupAndRestore_ButtonBackup.Content" xml:space="preserve">
+    <value>Backup</value>
+  </data>
+  <data name="General_SettingsBackupInfo_FileNameHeader.Text" xml:space="preserve">
+    <value>File name:</value>
+  </data>
+  <data name="General_SettingsBackupAndRestore_ButtonRefresh.Content" xml:space="preserve">
+    <value>Refresh</value>
+  </data>
+  <data name="General_SettingsBackupAndRestore_LinkRefresh.Text" xml:space="preserve">
+    <value>Refresh</value>
+  </data>
```

Comment:
Do we have a button or a link? Please remove the obsolete one?

Reply:
I believe that's the \<HyperLinkButton>